kairon/upload_handlers/definitions/factory.py

```
@@ -0,0 +1,23 @@
+        #UploadHandlerClass.broadcast: BroadcastFileUpload,
+        UploadHandlerClass.crud_data: CrudFileUploader
+    }
+
+    @staticmethod
+    def get_instance(upload_class: str):
+        """
+        Factory to retrieve uploadFile implementation for execution.
+
+        :param upload_class: valid uploadFile class
+        """
+        if upload_class not in UploadHandlerFactory.upload_handler.keys():
```

Ruff (0.12.2)

Use `key not in dict` instead of `key not in dict.keys()`

Remove `.keys()`

(SIM118)